### PR TITLE
Update companies.yml

### PR DESCRIPTION
**Pull request by lintqueen**
Deleted redhat (duplicate entry, should have a space); corrected entry for Red Hat, which is now required WFH (and added new reference URL)

Adds or updates the following companies, events, or universities:
 - 
 - 

#### Company updates
 - [x] I have put the most recent relevant date in the "Last Update" column
 - [x ] I have linked to the article about the change, not the company's homepage (Please put N/A if there is no public post)

#### Event updates
 - [ ] I have linked to the article about the change, not the event's homepage

#### University updates
 - [ ] I have linked to the article about the change, not the university's homepage

### CODE REVIEW

**Comment by lintqueen** [comment]
(this time I did it correctly, I think)

File: _data/companies.yml [hunk 1]
```diff
@@ -1659,11 +1659,11 @@
   last_update: 2020-03-04
 
 - name: Nosto
-  wfh: Mandatory
+  wfh: Encouraged
   travel: Restricted
   visitors: Restricted
   events: Restricted
-  last_update: 2020-03-13
+  last_update: 2020-03-10
```
Comment (drewbrew) [inline _data/companies.yml]: I think your GitHub client is trying to undo these two changes.

File: _data/companies.yml [hunk 2]
```diff
@@ -2000,13 +2000,6 @@
   visitors: "?"
   events: "?"
   last_update: 2020-03-04
-  
-- name: RedHat
-  wfh: Encouraged
-  travel: Restricted
-  visitors: "?"
-  events: Restricted
-  last_update: 2020-03-13
```
Comment (drewbrew) [inline _data/companies.yml]: I think your other PR that I merged took care of this.

File: _data/companies.yml [hunk 3]
```diff
@@ -2484,13 +2477,6 @@
   events: N/A
   last_update: 2020-03-13
 
-- name: TripActions
-  wfh: Required
-  travel: Restricted
-  visitors: Restricted
-  events: Canceled
-  last_update: 2020-03-12
-
```
Comment (drewbrew) [inline _data/companies.yml]: Is this intentional?

File: _data/companies.yml [hunk 4]
```diff
@@ -2616,13 +2602,6 @@
   visitors: "?"
   events: Restricted
   last_update: 2020-03-10
-  
-- name: Vouch
-  wfh: Required
-  travel: Suspended
-  visitors: Restricted
-  events: Restricted
-  last_update: 2020-03-12
```
Comment (drewbrew) [inline _data/companies.yml]: Is this intentional?

**Comment by krishnatre-siddhartha** [comment]
Trying to understand if #841 and #840 are same but done twice!

**Comment by krishnatre-siddhartha** [comment]
Trying to understand if #841 and #840 are same but done twice!
@drewbrew could we close both #840 and #841 as Red Hat is taken care of at red-hat.yml

**Comment by drewbrew** [comment]
Agreed, it looks like a duplicate to me. Feel free to reopen and fix the merge conflicts if this isn't the case.

File: _data/companies.yml (hunk 4 above)
Comment (lintqueen) [reply _data/companies.yml]: No idea why I only just now got this but, yeah, apologies all over. The step with the checkboexes threw me for a loop (don't github much) and I am pretty sure that I borked it then had to redo (at least once).

So very sorry.

**Pull request closed without merging**